source/_integrations/bluetooth_tracker.markdown

```diff
@@ -11,6 +11,8 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
+>**_NOTE:_** As of Core 2022.7 this integration does not work anymore for the installation methods Home Assistant Operating System, Container and Supervised due to the update to Python 3.10 (see [Breaking Changes](https://www.home-assistant.io/blog/2022/07/06/release-20227/#breaking-changes)).
```

Comment:
This is an issue that should be fixed, not documented.

Feel free to open up an issue in the core issue tracker or help out fixing the issue by opening a PR.

Thanks!

../Frenck